client.go

```diff
@@ -278,38 +280,38 @@ func (c *HelmClient) install(spec *ChartSpec) error {
 					RepositoryCache:  c.Settings.RepositoryCache,
 				}
 				if err := man.Update(); err != nil {
-					return err
+					return nil, err
 				}
 			} else {
-				return err
+				return nil, err
 			}
 		}
 	}
 
 	values, err := spec.GetValuesMap()
 	if err != nil {
-		return err
+		return nil, err
 	}
 
 	if c.linting {
 		err = c.lint(chartPath, values)
 		if err != nil {
-			return err
+			return nil, err
 		}
 	}
 
 	rel, err := client.Run(helmChart, values)
 	if err != nil {
-		return err
+		return nil, err
```

Comment:
We loose the information about the installed release in this case, if we just return the error.
It should be sufficient to just return all results of `client.Run` here.